CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ For information on changes in released versions of Teku, see the [releases page]
   - `beacon_eth1_current_period_votes_unknown` - Number of votes for blocks this node considers invalid (e.g. unknown blocks).
   - `beacon_eth1_current_period_votes_current` - Number of votes for the value in the current state (the default vote per the spec).
   - `beacon_eth1_block_cache_size` - Total number of blocks stored in the Eth1 block cache.
+- Added `migrate-data` subcommand to allow users to migrate to a leveldb relatively easily without requiring a full re-sync.   
 - Aggregate and batch verify signatures for gossiped attestations
 - Added `migrate-data` subcommand to allow users to migrate to a leveldb relatively easily without requiring a full re-sync.
```

Comment:
Seem to be trying to add the migrate-data entry twice.

Reply:
Got fixed as I was reviewing. :)

Reply:
ick merge. sorry